Dockerfile

```diff
@@ -128,7 +128,7 @@ ENV ANCHORE_CONFIG_DIR=/config \
     ANCHORE_OAUTH_TOKEN_EXPIRATION=3600 \
     ANCHORE_AUTH_ENABLE_HASHED_PASSWORDS=false \
     AUTHLIB_INSECURE_TRANSPORT=true \
-    ANCHORE_MAX_COMPRESSED_IMAGE_SIZE=null
+    ANCHORE_MAX_COMPRESSED_IMAGE_SIZE=-1
```

Comment:
Please add the unit here since it's not bytes, ANCHORE_MAX_COMPRESSED_IMAGE_SIZE_MB, will work. Just to be clear on units in the name.